### ISSUE TITLE
[Enhancement]: Playlist button for batch select

### ISSUE THREAD

**Issue by nanofred** [advplyr/audiobookshelf#1943]
### Describe the feature/enhancement

Collections has a button, and there's no way to add more than one book to a playlist at a time.

**Comment by GOvEy1nw** [comment]
Yes please, especially for podcast episodes, this would be great!

**Comment by damccull** [comment]
This would be useful in all locations that you can select more than one book simultaneously.

**Comment by advplyr** [comment]
Closing in favor of #3466
